packages/sources/snowflake/README.md

```diff
@@ -0,0 +1,95 @@
+# Chainlink External Adapter for Snowflake
+
+Adapter to query data from [Snowflake](https://www.snowflake.com/). Currently only contains an endpoint to query US confirmed Covid cases per County.
+To setup the environment, here are additional docs to [assign a key pair to a user](https://docs.snowflake.com/en/developer-guide/sql-api/guide.html#using-key-pair-authentication) and [finding your account identifier](https://docs.snowflake.com/en/user-guide/admin-account-identifier.html).
+
+### Environment Variables
+
+| Required? |    Name     |                                Description                                | Options |       Defaults to        |
+| :-------: | :---------: | :-----------------------------------------------------------------------: | :-----: | :----------------------: |
+|    ✅     | PRIVATE_KEY | Private PKCS PEM Key matching assigned public key for a user in Snowflake |         |                          |
+|    ✅     |   ACCOUNT   |                Unique identifier for the Snowflake account                |         |                          |
+|    ✅     | DB_USERNAME |                         Name of the database user                         |         |                          |
+|           |  DATABASE   |                    Name of the database to connect to                     |         | COVID19_BY_STARSCHEMA_DM |
+|           |   SCHEMA    |             Database schema where the main tables are located             |         |          PUBLIC          |
+|           |  PROVIDER   |           Shortcut for the selected cloud provider (e.g. 'aws')           |         |                          |
+|           |   REGION    |        Shortcut for the designated cloud region (e.g. 'us-east-2')        |         |                          |
```

Comment:
To be more explicit, do you think it would make sense to prefix these with `CLOUD_`?